src/EditorFeatures/VisualBasicTest/NavigateTo/NavigateToTests.vb

```diff
@@ -332,7 +332,7 @@ End Sub
 End Class", Async Function(w)
                 SetupVerifiableGlyph(StandardGlyphGroup.GlyphGroupMethod, StandardGlyphItem.GlyphItemPublic)
                 Dim item As NavigateToItem = (Await _aggregator.GetItemsAsync("Foo")).Single(Function(i) i.Kind = NavigateToItemKind.Method)
-                VerifyNavigateToResultItem(item, "Foo", MatchKind.Exact, NavigateToItemKind.Method, "New()", $"{FeaturesResources.type_space}Foo")
+                VerifyNavigateToResultItem(item, "Foo", MatchKind.Exact, NavigateToItemKind.Method, "Foo.New()", $"{FeaturesResources.type_space}Foo")
```

Comment:
I've gone with a different display for VB constructors.  This is necessary because we match teh pattern against the name ('Foo' in this case), which means that we'll just end up highlighting bogus entries in the display name.